python/src/Distribution.i

```diff
@@ -171,9 +179,15 @@ class SciPyDistribution(PythonDistribution):
         return rvs.reshape(size, 1)
 
     def computePDF(self, X):
-        # before 1.9, all 1d dist have a pdf method
-        # since 1.9, pdf for continuous, pmf for discrete
-        pdf = self._dist.pdf(X[0])
+        x0 = float(X[0])
+        if self.isContinuous():
+            pdf = self._dist.pdf(x0)
+        else:
+            if x0.is_integer():
+                pdf = self._dist.pmf(x0)
+            else:
+                # newer scipy pmf is a continuous function
```

Comment:
hmm suppose you have an equivalent of `UserDefined` with support points of type `(n+1)/2`, `n integer` (you are supposing discrete distribution + support is some `integer`...) 

Why don't we use `ppf` to get the support ? Seems that it is more robust than `pmf`

Reply:
I am usin ppf to get the support, see code above:
lb = dist.ppf(cdf_epsilon)
I also just saw that there's a "support" method that can be called instead of ppf, using that for the first call

For discrete but non-integral concern there is no api other than that, so currently I assume discrete means integral for now.
Browsing into scipy distributions I did not see any non-integral, but I may be wrong.

Anyway I added some comments for these aspects